README.md

```diff
@@ -56,10 +56,9 @@ Windows is not tested and not supported for now.
 
 - Create a local virtual environment with `virtualenv .` and activate this with `source bin/activate`
 - Install the required dependencies with `pip install -r requirements.txt`
-- If `scancode-toolkit==2.0.1` requirements gives an error
-    - cd into `scancode-toolkit` directory and install `scancode-toolkit==2.0.1` with `python setup.py install`
-- Comment `pytest-django` in requirements.txt in case of error
-- Install the requests library with `pip install requests`
+- If `scancode-toolkit==2.2.1` requirements gives an error
+    - comment the scancode-toolkit
+    - cd into `scancode-toolkit` directory and install `scancode-toolkit==2.2.1` with `python setup.py install`
```

Comment:
Now that you have updated the requirements, I guess none of this readme update is needed, right?

Reply:
Actually, if user has already installed the scancode-toolkit. Then the `scancode-toolkit==2.2.1` requirement will pass. That's why I left it there and updated readme rather. In case user hasn't installed then he/she will have to follow 60, 61 lines and then carry on with requirements.txt

Reply:
OK, remove these readme things then: they are useless when using the correct updated reqs.

Reply:
So, we have to consider that user has already installed scancode-toolkit and ready to proceed.